src/main/java/Duke.java

```diff
@@ -1,10 +1,122 @@
+import java.util.Scanner;
+
 public class Duke {
+    public static class Task {
+        protected String description;
+        protected boolean isDone;
+
+        public Task(String description) {
+            this.description = description;
+            this.isDone = false;
```

Comment:
"this" keyword should be utilised when the arguments are named similarly to the properties